.github/workflows/gradle.yml

```diff
@@ -31,5 +31,4 @@ jobs:
         distribution: 'temurin'
     - name: Setup Gradle
       uses: gradle/actions/setup-gradle@v4
-    - name: Build with Gradle
-      run: ./gradlew build
+    - run: ./gradlew build
```

Comment:
❓ Why the removal of the name?

Reply:
Oh that is just out of habit, happy to back out if you want.

It replaces the `Build with Gradle` text in the Action logs with the actual command... sou you immediately know what happens:
![image](https://github.com/user-attachments/assets/967a7ae0-4f56-4b61-a697-e80df6a2caa3)